### ISSUE TITLE
[Bug] Replace with title

### ISSUE THREAD

**Issue by dhruvwo** [darkreader/darkreader#9498]
# Bug Report
I am using npm library.
I am having an issue with rendering my webpage in a dark mode in Chrome only, It works perfectly fine in my local machine.
It works well with firefox and safari as well.

![Screenshot 2022-08-09 at 4 34 50 PM](https://user-images.githubusercontent.com/58214938/183632946-1c0465e4-5d77-49ed-9846-d7d2aee0c470.png)

## System Information

- OS: 
- Browser: 
- Dark Reader Version:

**Comment by Gusted** [comment]
Please try to look at [other](https://github.com/darkreader/darkreader/issues/3569) related [issues](https://github.com/darkreader/darkreader/issues/7274), it might contain valuable information.

Are you able to control the response headers of the requested styles?

**Comment by dhruvwo** [comment]
Yes, I am able to control the response headers.
We have set: "Access-Control-Allow-Origin": "*" in the response.
but still it is not working.

![image](https://user-images.githubusercontent.com/58214938/183825643-a8cd624b-b076-4f5e-86e4-d34c154b7cf2.png)

**Comment by dhruvwo** [comment]
I tried using the proxy as you suggested [here](https://github.com/darkreader/darkreader/issues/3569#issuecomment-685034821)
but getting 403.

![image](https://user-images.githubusercontent.com/58214938/183863328-79343991-2b8e-494e-954d-1bffd942deb9.png)
![image](https://user-images.githubusercontent.com/58214938/183863625-47d3b5aa-7640-4812-8927-f28874b91f72.png)

**Comment by dhruvwo** [comment]
yeah, I don't know why it is not working! and can't find a solution for that.
can you take a look at the demo URL I shared?
let me know if you wanna check the whole code, will share a git repo with you.

I am using this npm package in the react.js app.

here is the function which triggers when I toggle the "Dark Mode" switch.

`
          onChange={(checked) => {
            if (checked) {
              const defaultTheme = {
                brightness: 100,
                contrast: 90,
                sepia: 10,
              };
              const defaultFixes = {
                invert: [],
                css: '',
                ignoreInlineStyle: ['.react-switch-handle'],
                ignoreImageAnalysis: [],
                disableStyleSheetsProxy: true,
              };
              if (window.MutationObserver && window.fetch) {
                if (useSetFetch) {
                  console.log('setting setFetchMethod...', window.fetch);
                  const proxyFetch = (url) => {
                    console.log(
                      'settingFetch for',
                      url,
                      'with useProxyFetch',
                      useProxyFetch,
                      'and mode',
                      requestMode,
                    );
                    return window.fetch(
                      (useProxyFetch ? 'https://cors-anywhere.herokuapp.com/' : '') + url,
                      {
                        mode: requestMode,
                      },
                    );
                  };
                  if (useProxyFetch) {
                    const newScript = document.createElement('script');
                    newScript.src = 'https://unpkg.com/darkreader@latest/darkreader.js';
                    newScript.id = 'darkreaderId';
                    console.log('importing...', newScript);
                    document.body.appendChild(newScript);
                  }
                  setFetchMethod(proxyFetch);
                }
                console.log('will enable after ', setTimeoutForEnable, 'ms');
                setTimeout(() => {
                  console.log('enabling now...');
                  enable(defaultTheme, defaultFixes);
                }, setTimeoutForEnable);
              }
            } else {
              console.log('disabling...');
              disable();
            }
            setIsDarkMode(checked);
          }}`

**Comment by Jelmerro** [comment]
I have the same issue in my Electron app, even now that I have replaced `window.fetch` with the nodejs https module, (which shouldn't suffer from CORS issues,) I still see that the styling isn't applied, even though all the request are responded properly. If I disable web security on startup (using Chromium's `--disable-web-security` flag) I can see the styling being applied properly. I only noticed because a user of my app reported this issue, and made me resolve the CORS issues, but even now that they are solved, the styling still only works when disabling web security entirely. One strange thing I noticed, is that while all the requests are now green, there are a lot more being made when web security is disabled, which makes me think it's an issue with darkreader. Maybe the fetch method isn't used in all cases yet?
